massa-protocol-worker/src/protocol_network.rs

```diff
@@ -365,19 +358,28 @@ impl ProtocolWorker {
             serialized_data: content_serialized,
         };
 
+        let operation_set: Map<OperationId, usize> = info
+            .operations
+            .clone()
+            .unwrap()
+            .into_iter()
+            .enumerate()
+            .map(|(idx, op_id)| (op_id, idx))
+            .collect();
+
+        // Let the consensus store the block, operations and endorsements are already stored
+        // with [note_operations_from_node] and [note_endorsements_from_node]
+        let mut storage = self.storage.clone_without_refs();
```

Comment:
Related to my previous comment we can claim here to have in the storage

Reply:
can you recheck this with the latest conflict resolution ?